transbank/webpay/webpay_plus/transaction.py

```diff
@@ -75,6 +75,7 @@ def refund(cls, token: str, amount: float, options: Options = None):
         response = requests.post(url=endpoint, headers=HeadersBuilder.build(options),
                                  data=TransactionRefundRequestSchema().dumps(request).data)
         json_response = response.text
+        print(json_response)
```

Comment:
old school debugging mode mate?

Reply:
It's like... I dont know.. whatchamacallit... 
I will remove it.

Reply:
Done

Reply:
cheers!